draft-ietf-rats-architecture.md

```diff
@@ -1516,27 +1524,26 @@ The following example illustrates a hypothetical Background-Check Model solution
         |                |                |               |
         ~                ~                ~               ~
         |                |                |               |
-     time(HR_a)<---------+-------------time(HR_v)------>time(HR_r)
+     time(HR_a)<---------+-----------time(HR_v)----->time(HR_r)
         |                |                |               |
      time(EG_a)          |                |               |
-        |----Evidence{time(EG_a)}-------->|               |
-        | {Handle1,time(EG_a)-time(VG_a)}|                |
-        |                |             time(RG_v)         |
-        |<-----Attestation Result---------|               |
-        |   {time(RG_v),time(RX_v)}       |               |
-        |                |                                |
-        ~                ~                                ~
-        |                |                                |
-     time(HR_a')<--------'---------------------------->time(HR_r')
-        |                                                 |
-     time(RR_a)                                           /
-        |--Attestation Result{time(RX_v)-time(RG_v)}-->time(RA_r)
-        |    {Handle2, time(RR_a)-time(EG_a)}             |
-        ~                                                 ~
-        |                                                 |
-        |                                              time(OP_r)
-        |                                                 |
+        |---Evidence{H,time(VG_a)}------->|               |
+        |                |                |               |
+        |                |           time(RG_v)           |
+        |<--Attestation Result------------|               |
+        |   {H,time(RX_v)}                |               |
```

Comment:
time(RX_v) is useless in this message for the same reason as in my previous comment.